storage/volume/core_test.go

```diff
@@ -228,6 +228,54 @@ func TestAttachVolume(t *testing.T) {
 	// TODO
 }
 
+//Mock data for TestDetachVolume
+func mockVolumeCore(root string) (*Core, error) {
```

Comment:
this function is ok, but we already has a function to NewCore: `createVolumeCore`